turbinia/client.py

```diff
@@ -90,6 +93,66 @@
 logger.setup()
 
 
+def filter_jobs(jobs_blacklist, jobs_whitelist):
+  """Filter jobs based on provided whitelist/blacklist.
+
+  Args:
+    jobs_blacklist (Optional[list[str]]): Jobs we will exclude from running
+    jobs_whitelist (Optional[list[str]]): The only Jobs we will include to run
+  """
+  job_names = list(job_manager.JobsManager.GetJobNames())
+  jobs = []
```

Comment:
Maybe call this variable `jobs_remove` or something similar to be a little more clear?